lib/config/virtual_container_host.go

```diff
@@ -122,6 +122,7 @@ type Storage struct {
 	// Datastore URLs for image stores - the top layer is [0], the bottom layer is [len-1]
 	ImageStores []url.URL `vic:"0.1" scope:"read-only" key:"image_stores"`
 	// Permitted datastore URL roots for volumes
+
```

Comment:
fixed, that is odd... this was here before and I fixed it before. not sure why it came out.